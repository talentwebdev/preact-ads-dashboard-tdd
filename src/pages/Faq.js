import React from 'react'

function Faq() {
    return (
        <div>
            This is a Faq page
        </div>
    )
}

export default Faq
