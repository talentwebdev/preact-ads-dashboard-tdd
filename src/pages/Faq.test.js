import { shallow } from "enzyme";
import Faq from "./Faq";

describe("Faq", () => {
  let faqContainer;

  beforeEach(() => (faqContainer = shallow(<Faq />)));

  it("should render the div element", () => {
    expect(faqContainer.find("div")).toBeTruthy();
  });
  it("renders FAQ correctly", () => {
    expect(faqContainer).toMatchSnapshot();
  });
});
