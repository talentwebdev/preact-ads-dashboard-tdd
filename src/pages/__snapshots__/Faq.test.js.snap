// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Faq renders FAQ correctly 1`] = `ShallowWrapper {}`;
